dev_frankfurt.env

```diff
@@ -0,0 +1,58 @@
+PRINT_ENV=dev
+RANCHER_LABEL=dev
+IMAGE_TAG=staging
+NGINX_PORT=8009
+TOMCAT_PORT=8011
+API_URL=//mf-chsdi3.dev.bgdi.ch
+TOMCAT_SERVER_URL=//vpc-alb-bgdi-dev.intra.bgdi.ch
+PRINT_SERVER_HOST=service-print.dev.bgdi.ch
+PRINT_TEMP_DIR=/var/local/print
+PRINT_LOGLEVEL=10
+DEBUG=True
+APACHE_ENTRY_PATH=
+APACHE_BASE_PATH=main
+BRANCH_STAGING=
+GIT_BRANCH=
+MODWSGI_USER=www-data
+INSTALLDIR=/var/www/vhosts/mf-chsdi3/private/chsdi
+WSGI_APP=/var/www/vhosts/mf-chsdi3/private/chsdi/apache/application.wsgi
+KML_TEMP_DIR=/var/local/print/kml
+MODWSGI_CONFIG=production.ini
+ALTI_URL=//service-alti.dev.bgdi.ch
+APACHE_BASE_PATH=main
+APACHE_ENTRY_PATH=
+APACHE_LOG_LEVEL=debug
+APACHE_PORT=8009
+API_URL=//api3.geo.admin.ch
+AWS_DEFAULT_REGION=eu-central-1
+CACHE_CONTROL=no-cache
+CMSGEOADMINHOST=https://cms.geo.admin.ch
+DATAGEOADMINHOST=data.geo.admin.ch
+DBHOST=pg.bgdi.ch
```

Comment:
here we should switch to `pg-geodata-replica.bgdi.ch` which is only available in frankfurt